ibc_test.go

```diff
@@ -105,17 +105,21 @@ func TestIBCHandshake(t *testing.T) {
 	init_msg := IBCInstantiateMsg{
 		ReflectCodeID: REFLECT_ID,
 	}
-	ires, _, err := vm.Instantiate(checksum, env, info, toBytes(t, init_msg), store, *goapi, querier, gasMeter1, TESTING_GAS_LIMIT, deserCost)
+	i, _, err := vm.Instantiate(checksum, env, info, toBytes(t, init_msg), store, *goapi, querier, gasMeter1, TESTING_GAS_LIMIT, deserCost)
 	require.NoError(t, err)
+	assert.NotNil(t, i.Ok)
+	ires := i.Ok
```

Comment:
The outer type is called "result" now and the inner one is "response". So "res" is not a great name. Could you rename those to `iResponse` and friends?

Reply:
Sure, will do. I was just following the Packet response test naming. Will rename that one too

Reply:
Yeah there is a bit of legacy naming I did not invent. Good we have the opportunity to improve those things now.